ui/apps/platform/src/Containers/Policies/Table/PoliciesTablePage.tsx

```diff
@@ -178,6 +186,37 @@ function PoliciesTablePage({
 
     return (
         <>
+            <PageTitle title="Policy management" />
```

Comment:
In analogy with Access Control, how about `title="Policy Management - Policies"` prop for plural entity and then edit the following
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Policies/PolicyPage.tsx#L147
to become `title="Policy Management - Policy"`

Then the updated property in `basePathToLabel` becomes a fallback just in case.

As you continue, there will be an additional pair of titles, something like:
* Policy Management - Categories
* Policy Management - Category